api/api_sources/sources/server/modules/location/location.route.ts

```diff
@@ -73,8 +71,8 @@ export class LocationRouteController extends RouteController {
         ],
         secure: true
     })
-    get wells(): RouteHandler {
-        return this.routeConfig<any>(`${this.className}: codes`, async (d: any, req: any) => [200, await this.fetchWellsData(req.query)]);
+    async wells(req: any) {
```

Comment:
can we rename this function to clarify what it's doing? e.g. it's not immediately clear what the diff is between this function and fetchWellsData()